hoot-core-test/src/test/cpp/hoot/core/io/OsmApiDbTest.cpp

```diff
@@ -71,23 +84,36 @@ class OsmApiDbTest : public CppUnit::TestFixture
     }
   }
 
-  QUrl getOsmApiDbUrl()
+  //TODO: the sql output shouldn't be written out to null; simply checking the return value
+  //isn't enough to know whether it failed or not
```

Comment:
That actually did help catch the problem that was silently logging an error when running those sql scripts before (missing env vars).  thanks.  added the change.  The test has other checks as well on the sql query results that will reveal that problem, but that change catches it even earlier...so a good one.

Also just pushed some other changes that fix the writing of the changeset as sql.